Sliding Window/Find Maximum in Sliding Window in Java

```diff
@@ -0,0 +1,58 @@
+import java.util.*;
```

Comment:
The code looks good, you can remove iml file

Reply:
I have removed iml file could you please review it